packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -571,64 +501,91 @@ describe('ConnectionFlow', () => {
       )
     })
 
-    it('should return unmodified account trigger folder does not exist', async () => {
+    it('should return unmodified account trigger when folder does not exist', async () => {
       const { flow, client } = setup()
       prepareTriggerAccount.mockResolvedValue(fixtures.updatedAccount)
 
       ensureTrigger.mockResolvedValue(
         fixtures.createdTriggerWithFolder.attributes
       )
       client.query.mockRejectedValue(new Error('404'))
-      await flow.ensureTriggerAndLaunch(client, {
+      await setupSubmit(flow, {
         account: fixtures.existingAccount,
         trigger: fixtures.existingTrigger,
         konnector: fixtures.konnectorWithFolder
       })
       expect(flow.account).toEqual(fixtures.updatedAccount)
-      expect(saveAccount).not.toHaveBeenCalled()
+      expect(saveAccount).toHaveBeenCalledTimes(1)
       expect(client.query).toHaveBeenCalledWith(
         Q('io.cozy.files').getById(
           fixtures.createdTriggerWithFolder.attributes.message.folder_to_save
         )
       )
     })
+    delete window.cozy
+    delete window.ReactNativeWebView
```

Comment:
Where are these variables defined? Is it because of https://github.com/cozy/cozy-libs/pull/2000/commits/9f251f802b0dd046d1e99768b3b402350baed1b8#diff-57ee38fdf45b61888d99d54f38f4dfe8bc42c88c8e3af8e1026d120a1b9af17fR407-R411 ? 

If yes, this is a red flag. Each test must be autonomous and have no side effect

Reply:
Right, I put it back inside the same test now